src/rust/ide/view/graph-editor/src/component/visualization/container/fullscreen.rs

```diff
@@ -0,0 +1,99 @@
+//! A module containing the fullscreen view of visualization.
+
+use crate::prelude::*;
+
+use ensogl::data::color;
+use ensogl::display;
+use ensogl::display::DomSymbol;
+use ensogl::display::scene::Scene;
+use ensogl::display::shape::*;
+use ensogl::display::traits::*;
+use ensogl::system::web;
+use ensogl::system::web::StyleSetter;
+use ensogl_theme as theme;
+
+
+
+// ==============
+// === Shapes ===
+// ==============
+
+/// Container background shape definition.
+///
+/// Provides a backdrop and outline for visualisations. Can indicate the selection status of the
+/// container.
+/// TODO : We do not use backgrounds because otherwise they would overlap JS
+///        visualizations. Instead we added a HTML background to the `View`.
+///        This should be further investigated while fixing rust visualization displaying. (#526)
+pub mod background {
+    use super::*;
+
+    ensogl::define_shape_system! {
+        (style:Style,selected:f32,radius:f32,roundness:f32) {
+            let width  : Var<Pixels> = "input_size.x".into();
+            let height : Var<Pixels> = "input_size.y".into();
+            let radius        = 1.px() * &radius;
+            let color_path    = theme::graph_editor::visualization::background;
+            let color_bg      = style.get_color(color_path);
+            let corner_radius = &radius * &roundness;
+            let background    = Rect((&width,&height)).corners_radius(&corner_radius);
+            let background    = background.fill(color::Rgba::from(color_bg));
+            background.into()
+        }
+    }
+}
+
+
+
+// ======================
+// === FullscreenView ===
+// ======================
+
+/// View of the visualization container meant to be used in fullscreen mode. Its components are
+/// rendered on top-level layers of the stage.
```

Comment:
I dont think this is strictly true. Components of full screen visualization panel, like HTML visualizations will not be rendered on top-level layers of the stage, will they be?